src/conspiracies/docprocessing/docprocessor.py

```diff
@@ -25,7 +25,12 @@ def warn_error(proc_name, proc, docs, e):
         return nlp_coref
 
     def _build_triplet_extraction_pipeline(self):
-        nlp = spacy.load("da_core_news_sm")
+        if self.language == "da":
+            nlp = spacy.load("da_core_news_sm")
+        elif self.language == "en":
+            nlp = spacy.load("en_core_web_sm")
+        else:
+            raise ValueError(f"Unsupported language: {self.language}")
```

Comment:
I would just allow people to specify the model to use here.

Reply:
Could be a configuration option, but there should also be a default based on language choice.